src/simulators/matrix_product_state/matrix_product_state_internal.hpp

```diff
@@ -261,7 +274,7 @@ class MPS{
   uint_t omp_threads_ = 1;     // Disable multithreading by default
   uint_t omp_threshold_ = 14;  // Qubit threshold for multithreading when enabled
   int sample_measure_index_size_ = 10; // Sample measure indexing qubit size
-  double json_chop_threshold_ = 0;  // Threshold for choping small values
+  double json_chop_threshold_ = 1E-8;  // Threshold for choping small values
```

Comment:
This is larger than the value in `matrix_product_state.hpp` (1e-15). Can you set them both to the same value? I think 1e-8 is also a little large, the other state classes use 1e-10.

Reply:
@chriseclectic , you are right. I have some old code here that was originally copied from one of the other simulation methods. I was just looking at this in the context of the approximation issue. I intend to set the default for chopping the Schmidt coefficient to 1e-16, and this will be controllable by the user. However, I prefer to handle in the `approx` issue, and not here. Let me know if this sound like a reasonable default threshold.